_posts/2018/2018-04-15-use-hash-fetch-when-using-params-in-controllers.md

```diff
@@ -0,0 +1,71 @@
+---
+title: 'Use Hash#fetch when using Rails params in controllers'
+description: ''
+layout: article
+category: ruby
+image:
+  base: '2018/use-hash-fetch-when-using-params-in-controllers'
+  alt: 'Retriever'
+  source: 'https://unsplash.com/photos/Pzu9f6Nby5w'
+  credit: 'Mitchell Orr'
+---
+
+We often need to access a parameter from the URL that isn't part of the regular rails routes. Something passed after
```

Comment:
What are you trying to get across with this sentence? Why are you starting with a URL parameter as an example? Is that the best one?

(Since you're giving me unfinished first paragraphs, might was well give you a hard time via Q&A. 😂)